test_sample.py

```diff
@@ -1,6 +1,94 @@
 # content of test_sample.py
-def func(x):
-    return x + 1
+import subprocess as subprocess
+import os
+import requests
+
+def getLanguages():
+  	URL = "http://104.248.116.233:5000/language-list"
```

Comment:
We should be able to in the future for sure, but currently because the data is all in the main server, I think for now it makes sense to just leave this URL. Does that make sense to you?